azure-pipelines.yml

```diff
@@ -0,0 +1,77 @@
+name: k8sUIWebApp_$(SourceBranchName)_$(Date:MMMd)$(Rev:.r)
+
+# PR trigger
+pr:
+- master
```

Comment:
I had earlier edited the yaml file in mseng to include PR trigger against releases/* branches as well. plz do the same here as well.